src/components/LogOutput/LogOutput.test.ts

```diff
@@ -0,0 +1,35 @@
+import { describe, it, expect } from 'vitest';
+import { ref } from 'vue';
+import { mount } from '@vue/test-utils';
+import LogOutput from '@/components/LogOutput/LogOutput.vue';
+import { Vigad } from '@/proc/Vigad';
+
+describe('CaptureAreaSearchValue', () => {
+    /**
+     * Vigad instance
+     */
+    const vigad = ref(Vigad.getInstance());
+
+    /**
+     * Add capture area
+     */
+    vigad.value.addCaptureArea(100, 100, 0, 0);
+
+    /**
+     * Capture areas
+     */
+    const captureAreas = ref(vigad.value.getAllCaptureAreas());
+
+    const props = {
+        captureAreaId: captureAreas.value[0].getId(),
+    };
+
+    it('displays "Start capturing first!" when matchedElements is empty', async () => {
+        const wrapper = mount(LogOutput, {
+            props: props,
+        });
+        const message = wrapper.find('p');
+        expect(message.exists()).toBe(true);
+        expect(message.text()).toBe('Start capturing first!');
```

Comment:
I feel like it's kind of bad practice to test if a specific string literal is shown to the user (as this is something that is expected to change). I think testing if the message exists is sufficient here